src/averell/core.py

```diff
@@ -1,29 +1,57 @@
 import importlib
+import logging
+import os
 from pathlib import Path
 
-import yaml
-
-from averell.utils import download_corpora
-from averell.utils import filter_features
+from .utils import CORPORA_SOURCES
+from .utils import download_corpora
+from .utils import filter_features
+from .utils import write_json
 
 DEFAULT_OUTPUT_FOLDER = Path.cwd() / "corpora"
 
-CORPORA_SOURCES = yaml.load("corpora.yaml", Loader=yaml.FullLoader)
-
 
 def get_corpora(corpus_indices=None, granularity=None,
                 output_folder=DEFAULT_OUTPUT_FOLDER):
     """
 
+    :param corpus_indices:
```

Comment:
Please add a basic description of the variables

Reply:
Done